flink-test-utils-parent/flink-connector-testing/src/main/java/org/apache/flink/connectors/test/common/external/sink/TestingSinkOptions.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.connectors.test.common.external.sink;
+
+import org.apache.flink.connector.base.DeliveryGuarantee;
+
+/** Options for configuring the sink under testing. */
+public class TestingSinkOptions {
```

Comment:
`TestingSinkOptions` only contains one field and is not flexible, we event can not add new configuration without change this class in the future